packages/gatsby-transformer-pdf/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "gatsby-transformer-pdf",
+  "description": "Gatsby transformer plugin for pdf",
+  "version": "2.0.0",
+  "author": "Alex Munoz <almunozdo@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.0.0",
```

Comment:
Is this needed as a dep?

Reply:
i followed the structure of the others plugins. I'm not completely sure about this dependency :(